checks/branch_protected.go

```diff
@@ -54,55 +55,87 @@ func IsBranchProtected(protection *github.Protection, c *checker.CheckRequest) c
 	totalChecks := 6
 	totalSuccess := 0
 
-	if protection.GetAllowForcePushes() != nil {
-		if protection.AllowForcePushes.Enabled {
-			c.Logf("!! branch protection AllowForcePushes enabled")
-		} else {
-			totalSuccess++
-		}
+	// This is disabled by default (good).
+	if protection.GetAllowForcePushes() != nil &&
+		protection.AllowForcePushes.Enabled {
+		c.Logf("!! branch protection AllowForcePushes enabled")
```

Comment:
maybe improve language

branch protection AllowForcePushes should not be enabled
same for others.